packages/fontaine/src/index.ts

```
@@ -1,7 +1,72 @@
+ */
+export const generateFallbackCSS = (fontName, options) => {
+    return generateFallbackCSS(fontName, options);
+};
+
+/**
+ * Transforms font metrics to a standard scale, useful for runtime or build-time calculations.
+ * @param {string} text The text input to measure.
+ * @param {Object} options Options for transformation.
+ * @returns {number} The scaled metric value.
+ */
+export const transformTextMetrics = (text, options) => {
```

ESLint

[error] 'transformTextMetrics' is already defined.

(ts/no-redeclare)